internal/daemonset/daemonset.go

```diff
@@ -310,3 +322,44 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 
 	return labels
 }
+
+func MakeLoadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	loadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
+		return append(loadCommand, ra.Load...)
+	}
+
+	if a := spec.Args; a != nil && len(a.Load) > 0 {
```

Comment:
Why do we need to check that len(ra.Load)>0 if it is required in the spec that minItems=1?

Reply:
Same reason as above.